spec/active_record/connection_adapters/makara_mysql2_adapter_spec.rb

```diff
@@ -3,10 +3,10 @@
 
 describe 'MakaraMysql2Adapter' do
 
-  let(:db_username){ ENV['TRAVIS'] ? 'travis' : 'root' }
+  let(:db_username){ ENV['TRAVIS'] ? 'travis' : ENV['MYSQL_USER'] || 'root' }
```

Comment:
yeah. I had simply converted the original default of `'root'`, to using an envar, with a default backup value.

I'm not sure if the `TRAVIS` envar should override the `MYSQL_USER` or not.